SCClassLibrary/Common/Streams/ListPatterns.sc

```diff
@@ -391,7 +391,7 @@ Pwalk : ListPattern {
 
 
 	embedInStream { arg inval;
-		var	step;
+		var	step, stepDir;
```

Comment:
"stepDir" sounds like "step direction", which it isn't. Maybe "dirStep" or "directionalStep"?

Reply:
OK, done.